docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
```

Comment:
I think we can drop this heading and just let the paragraph below stand on its own.